plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/functions/tablechanges/TableChangesFunction.java

```diff
@@ -100,9 +105,38 @@ public TableFunctionAnalysis analyze(
         long sinceVersion = -1; // -1 to start from 0 when since_version is not provided
         if (sinceVersionValue != null) {
             sinceVersion = (long) sinceVersionValue;
-            checkFunctionArgument(sinceVersion >= 0, "Invalid value of since_version: %s. It must not be negative.", sinceVersion);
+            checkFunctionArgument(sinceVersion >= 0 || sinceVersion == DEFAULT_GUARD_VALUE, "Invalid value of since_version: %s. It must not be negative.", sinceVersion);
+        }
+
+        ScalarArgument sinceTimestampArgument = (ScalarArgument) arguments.get(SINCE_TIMESTAMP_ARGUMENT);
+        Object sinceTimestampValue = sinceTimestampArgument.getValue();
+        long sinceTimestamp = 0L;
```

Comment:
`L` is redundant. `0` is a bad default value, since timestamps go into negative values.
obviosuly delta table format didn't exist before 1970, but we shouldn't fail or do anything stupid if user asks us for changes starting from 1953. 

Also, even though Delta table format didn't exist before 1970, someone can synthesize transaction log entries declaring dates before 1970 and we should respect that.